src/ui/mainwindow.cpp

```diff
@@ -522,13 +522,12 @@ QList<QAction*> MainWindow::getActions() const
 
 void MainWindow::setupLanguagesMenu()
 {
-    //ui->menu_Language->setStyleSheet("* { menu-scrollable: 1 }");
     std::map<QChar, QMenu*> menuInitials;
     for (const auto& l : LanguageCache::getInstance().languages()) {
         QString id = l.id;
         QChar letter = l.name.isEmpty() ? '?' : l.name.at(0).toUpper();
         QMenu *letterMenu;
-        if (menuInitials.count(letter)) {
+        if (menuInitials.contains(letter)) {
```

Comment:
`std::map` doesn't have `contains` (an oversight if you ask me). `count` is fine but explicitly written as `menuInitials.count(letter) > 0` so C++ is not implicitly casting 0/1 to true/false.